eureka/tests/NIRCam_ecfs/S5_fit_par.ecf

```diff
@@ -13,3 +13,8 @@ u1			0.3				'free'			-1				1				U
 u2			0.1				'free'			-1				1				U
 transittype	'primary'	 	'independent'
 c0			1.009			'free'			0.95			1.05			U
+#GP parameters
+A			-5.			'free'			-15.			5.			U
```

Comment:
These parameters likely need to be commented out in all the ecfs unless GP is in run_myfuncs, right? Otherwise the fitter will try to fit these unused parameters which will slow things down for no reason